.circleci/config.yml

```diff
@@ -1591,6 +1591,30 @@ workflows:
             - attach_workspace:
                 at: /
 
+  GCP-Daily-Services-Query-Restart-Performance-6N-6C:
+    triggers:
+      - schedule:
+          cron: "0 6 * * 2,6"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - build-platform-and-services
+      - jrs-regression:
+          context: Slack
+          regression_path: /swirlds-platform/regression
+          result_path: results/6N_6C/Restart
```

Comment:
nit : As mentioned in regression PR , it will be easy to differentiate if results folder is `results/6N_6C/QueryPerf` from Restart tests.

Reply:
Changed.